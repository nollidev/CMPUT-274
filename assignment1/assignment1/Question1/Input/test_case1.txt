mscott@gmail.com jhalpert@domain.ca dschrute@gmail.com $km@gmail.com pbeesly@spam.ca