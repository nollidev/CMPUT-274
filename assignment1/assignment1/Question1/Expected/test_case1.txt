Valid
Valid
Valid
Invalid
Forbidden
